src/components/FriendPage/Modal.tsx

```diff
@@ -0,0 +1,70 @@
+import styled from "styled-components";
+import userData from "../../json/userData.json"
+
+interface ModalProps {
+  userId: number;
+  closeModal: () => void;
+}
+
+const Modal = ({ userId, closeModal }: ModalProps) => {
+  const user = userData.users.find((u) => u.id === userId);
+
+  return (
+    <ModalOverlay>
+      <ModalWrapper>
+        <button onClick={closeModal}>X</button>
+        <ModalContent>
+          <ProfileImg>
+            <img src={user?.profileImage}></img>
+          </ProfileImg>
+          <h1 style={{marginBottom: "5px"}}>{user?.name}</h1>
+          <h3>{user?.message}</h3>
+        </ModalContent>
+      </ModalWrapper>
+    </ModalOverlay>
+  );
+};
+
+export default Modal;
+
+const ModalOverlay = styled.div`
+  position: fixed;
+  top: 0;
+  left: 3.5rem;
```

Comment:
현재 모달 background 왼쪽 영역에 빈 부분이 생기는데 아래와 같이 처리해주면 될 것 같아요! 
```suggestion
  left: 0;
```